tldraw/tldraw
# Stop editing frame headers when clicking inside a frame.

Before (notice how we stay in `editing_shape`:

https://github.com/tldraw/tldraw/assets/2523721/d4553675-8412-41d9-b300-bbe87b719caa

After:


https://github.com/tldraw/tldraw/assets/2523721/e9788b1c-f2d9-473d-ba4c-cec7f822547a

It selects the shape, similar to what we do with geo shapes:


https://github.com/tldraw/tldraw/assets/2523721/a6210577-aec0-4fc0-892a-eedfed6f737b


The only downside is that we also stop editing shape if we click on the frame header. That said that case is currently a bit broken. We'd probably want to insert caret at the clicked position (which we currently don't do).

### Change Type


### Test Plan

1. Insert a frame.
2. Double click the frame header to edit it.
3. Click inside the frame. 
4. This should now stop editing the header.


Closes #1965 

### Release Notes

- Stop editing frame headers when clicking inside of a frame.
